client/me/purchases/purchases-list-in-dataviews/purchases-data-view.tsx

```diff
@@ -0,0 +1,43 @@
+import { Card } from '@automattic/components';
+import { Purchases } from '@automattic/data-stores';
+import { DataViews, View } from '@wordpress/dataviews';
+import { LocalizeProps } from 'i18n-calypso';
+import { usePurchasesFieldDefinitions } from './hooks/use-field-definitions';
+
+export const purchasesDataView = {
+	type: 'table',
+	page: 1,
+	perPage: 5,
+	titleField: 'site',
+	fields: [ 'product' ],
+	layout: {},
+} as View;
+
+export function PurchasesDataViews( props: {
+	purchases: Purchases.Purchase[];
+	translate: LocalizeProps[ 'translate' ];
+} ) {
+	const { purchases } = props;
+	const onChangeView = () => {
+		alert( 'You clicked something!!' );
+	};
+
+	const getItemId = ( item: Purchases.Purchase ) => {
+		return item.id.toString();
+	};
+	const purchasesDataFields = usePurchasesFieldDefinitions();
+	return (
+		<Card id="purchases-list" className="section-content" tagName="section">
```

Comment:
I am getting a strange error here: 

> 'React' refers to a UMD global, but the current file is a module

It looks like it is related to how TypeScript is configured. Something seems to be missing so it expects me to import React
- I rebased against a fresh pull of Automattic/wp-calypso trunk
- I checked the Git Blame for client/tsconfig.json, and it doesn't look like anything changed there.
- I'll check my `.gitignore`

Reply:
🤔 I don't see that warning either in my editor's TypeScript LSP nor when running `yarn typecheck`. Where do you see it?